test/Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests/SchemaTests/JSONSchemaTests.cs

```diff
@@ -44,8 +44,12 @@ public void IsJSONSchemaValid(string testFile)
 
         public static IEnumerable<object?[]> GetAllTemplates()
```

Comment:
Should we run thouse through `ValidateAsync` as well (skipping the localization checking)?

Reply:
added, nice idea! once rebased, we should also add tests for samples.